generators/app/templates/src/app/components/CustomTextInput/components/ShowPassword/styles.js

```diff
@@ -0,0 +1,16 @@
+import { StyleSheet } from 'react-native';
+
+import { blue } from '../../../../../constants/colors';
+
+export default StyleSheet.create({
+  container: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    bottom: -4
```

Comment:
Mmm, why this ? This doesn't have issues with `Android` overflow ?
If so, I'd put `marginBottom = -4`